core/drivers/clk/sam/at91_clk.h

```diff
@@ -186,6 +203,17 @@ at91_clk_register_master_div(struct pmc_data *pmc,
 			     const struct clk_master_layout *layout,
 			     const struct clk_master_charac *charac);
 
+/*
+ * @mux_table: when @mux_table is not NULL it shall hold @num_prents cells
```

Comment:
s/@num_prents/@num_parents/

Reply:
Updated and added `Acked-by`, thank you.